signing/sigstore/rekor.go

```diff
@@ -78,6 +78,7 @@ func (r *Rekor) Upload(ctx context.Context, att signing.Attestation) (signing.Lo
 	}
 	var uuid string
 	for ix, entry := range resp.Payload {
+		entry := entry
```

Comment:
![image](https://user-images.githubusercontent.com/172697/175564515-741b924f-10bb-42cb-84e2-98518406aeee.png)

https://github.com/securego/gosec#available-rules

Reply:
Yah, I had guesssed it was G601. I'm just not sure why it's an issue in this case. It's usually an issue when goroutines are involved because the goroutines [race over the value](https://github.com/golang/go/wiki/CommonMistakes#using-goroutines-on-loop-iterator-variables).

I'm not against adding code like this but it kind of stands out and I'd like to add a comment that explains the reasoning for why it's there. The docs seem pretty sparse on the rationale and recommendations so I'm not sure if there are other reasons for it.
https://securego.io/docs/rules/rule-intro.html

Reply:
After some thought, I agree. I will remove it. Thanks!